index.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link  href="css/style.css" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <title>Zuhra Hashimi - Portfolio</title>
+</head>
+
+<body>
+    <div>
+    <section>
+        <header>
+            <div>
+            <nav class="topnav">
+            
+                <a href="#home" class="logo">Zuhra Hashimi</a>
+                <div class="hamburger">
+                    <span class="bar"></span>
+                    <span class="bar"></span>
+                    <span class="bar"></span>
+                </div>
+            </nav>
+        </div>
+        </header>
+    </section></div>
+
+    <section class="main-content">
+        <div class="container">
+            
+            <h1 class="container headline">Hey There.<br>I'm Zuhra "Hashimi"</h1>
+            <h2 class="container job-title">I am a web full stack developer </h2>
+            <p class="container job-description">I can help you build a product, feature or website.Look through some of my work and experience! If you like what you see and have a project you need coded, do not hestiate to contact me.</p>
+        </div>
+    
+        <div class="links">
+            <ul class="social_media">
+                <li>
+                    <a href="https://github.com/ZuhRa-HashiMi" id="git" target="blank"><img src="assets/github.svg" alt="github"></a>
+                </li>
+                <li>
+                    <a href="https://www.linkedin.com/in/zuhra-hashimi-601966214/" id="linkedin" target="blank"><img src="assets/linkedin.svg" alt="linkedin"></a>
+                </li>
+             
+                <li>
+                    <a href="https://twitter.com/ZuhraHashimi4" id="twitter" target="blank"><img src="assets/twitter.svg" alt="twitter"></a>
+                </li>
+                <li>
+                    <a href="http://medium.com" id="medium" target="blank"><img src="assets/medium.svg" alt="linkedin"></a>
+                </li>
+                <li>
+                    <a href="http://angellist.com" id="angel-list" target="blank"><img src="assets/angellist.svg" alt="angel-list"></a>
+                </li>
+               
+                
+                
+                
```

Comment:
- [OPTIONAL] Kindly remove this empty lines because they are not making your code clean. let's do that and make your code looks professional.